bin/smarthome.py

```diff
@@ -131,7 +132,7 @@ class SmartHome():
     _dbapis = {}
     logger = logging.getLogger(__name__)
 
-    def __init__(self, smarthome_conf_basename=BASE + '/etc/smarthome'):
+    def __init__(self, smarthome_conf_basename = os.path.join(BASE + '/etc/smarthome'.replace('/', os.path.sep))):
```

Comment:
Could the default configuration path be moved to a constant into `constants.py`?